packages/yourloops/components/switch-role/index.tsx

```diff
@@ -29,50 +29,57 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
 import bows from "bows";
+import _ from "lodash";
 
 import metrics from "../../lib/metrics";
 import { useAuth } from "../../lib/auth";
 import { useAlert } from "../utils/snackbar";
 import { SwitchRoleDialogsProps, SwitchRoleToHcpSteps } from "./models";
 import SwitchRoleConsequencesDialog from "./consequences-dialog";
 import SwitchRoleConsentDialog from "./consent-dialog";
+import { Profile } from "models/shoreline";
+import { HcpProfession } from "models/hcp-profession";
```

Comment:
For the import it's always better to specify the path.
If not, sometimes we have some trouble, because it will search in `node_modules/models/shoreline` 
Also models, are higher level components, it should just before the metrics one.